openedx_external_enrollments/models.py

```diff
@@ -38,3 +40,26 @@ class Meta(object):
         Model meta class.
         """
         app_label = "openedx_external_enrollments"
+
+
+class OtherCourseSettings(models.Model):
+    """
+    Model to persist other course settings.
+    """
+
```

Comment:
I think this blank line is not necessary. @ivanvgh

Reply:
Removed